Module not found: Error: Cannot resolve module 'react-textarea-autosize' when 'npm start`

When running ```npm start``` at the bottom of the log, I see following errors.

```
  [601] ./app/js/components/users/sign-in.jsx 5.16 kB {1} [built]

ERROR in ./app/js/components/posts/PostDetail.jsx
Module not found: Error: Cannot resolve module 'react-textarea-autosize' in /Users/barton/projects/Serverless/discuss/client/src/app/js/components/posts
 @ ./app/js/components/posts/PostDetail.jsx 37:29-63

ERROR in ./app/js/components/comments/Comment.jsx
Module not found: Error: Cannot resolve module 'react-textarea-autosize' in /Users/barton/projects/Serverless/discuss/client/src/app/js/components/comments
 @ ./app/js/components/comments/Comment.jsx 33:29-63
webpack: bundle is now VALID.
```

Subsequently if I continue, when I try to hit ```http://localhost:8080``` in the ```console log``` I see:

```
/app/js/components/posts/PostDetail.jsx
Module not found: Error: Cannot resolve module 'react-textarea-autosize' in /Users/barton/projects/Serverless/discuss/client/src/app/js/components/posts
resolve module react-textarea-autosize in /Users/barton/projects/Serverless/discuss/client/src/app/js/components/posts
  looking for modules in /Users/barton/projects/Serverless/discuss/client/src/node_modules
    /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize doesn't exist (module as directory)
    resolve 'file' react-textarea-autosize in /Users/barton/projects/Serverless/discuss/client/src/node_modules
      resolve file
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.webpack.js doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.web.js doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.js doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.jsx doesn't exist
  looking for modules in /Users/barton/projects/Serverless/discuss/node_modules
    /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize doesn't exist (module as directory)
    resolve 'file' react-textarea-autosize in /Users/barton/projects/Serverless/discuss/node_modules
      resolve file
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.webpack.js doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.web.js doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.js doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.jsx doesn't exist
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.webpack.js]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.web.js]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.js]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.jsx]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.webpack.js]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.web.js]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.js]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.jsx]
 @ ./app/js/components/posts/PostDetail.jsx 37:29-63errors @ app.js:641
app.js:641 ./app/js/components/comments/Comment.jsx
Module not found: Error: Cannot resolve module 'react-textarea-autosize' in /Users/barton/projects/Serverless/discuss/client/src/app/js/components/comments
resolve module react-textarea-autosize in /Users/barton/projects/Serverless/discuss/client/src/app/js/components/comments
  looking for modules in /Users/barton/projects/Serverless/discuss/client/src/node_modules
    resolve 'file' react-textarea-autosize in /Users/barton/projects/Serverless/discuss/client/src/node_modules
      resolve file
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.webpack.js doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.web.js doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.js doesn't exist
        /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.jsx doesn't exist
    /Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize doesn't exist (module as directory)
  looking for modules in /Users/barton/projects/Serverless/discuss/node_modules
    resolve 'file' react-textarea-autosize in /Users/barton/projects/Serverless/discuss/node_modules
      resolve file
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.webpack.js doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.web.js doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.js doesn't exist
        /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.jsx doesn't exist
    /Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize doesn't exist (module as directory)
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.webpack.js]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.web.js]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.js]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize.jsx]
[/Users/barton/projects/Serverless/discuss/client/src/node_modules/react-textarea-autosize]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.webpack.js]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.web.js]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.js]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize.jsx]
[/Users/barton/projects/Serverless/discuss/node_modules/react-textarea-autosize]
 @ ./app/js/components/comments/Comment.jsx 33:29-63errors @ app.js:641
````

--- a/client/src/package.json
+++ b/client/src/package.json
@@ -42,6 +42,7 @@
     "react-redux": "^4.4.5",
     "react-router": "^2.4.0",
     "react-router-redux": "^4.0.4",
+    "react-textarea-autosize": "^4.0.3",
     "react-timeago": "^3.0.0",
     "redux": "^3.5.2",
     "redux-logger": "^2.6.1",

